website/docs/migration/migration-from-yoga-v1.mdx

```diff
@@ -7,9 +7,9 @@ type: Guide
 
 ## Installation
 
-You can start with installing `@graphql-yoga/node` package.
+You can start with updating `graphql-yoga` package.
```

Comment:
We also need a migration from Yoga v2 page

Reply:
Yes we definitely need to have that but I think we can start writing that page after finishing all the remaining tasks. Because there will be more changes in the API.

Reply:
But then we should not update existing docs as those are the old v2 docs that we still should keep around?

Reply:
Ok I moved the content of the changeset to the migration guide draft so we can gradually update it.